apps/studio/src/lib/db/clients/mysql.js

```diff
@@ -722,6 +724,15 @@ export async function truncateAllTables(conn) {
   });
 }
 
+export async function dropElement (conn, elementName, typeOfElement) {
+  await runWithConnection(conn, async (connection) => {
+    const connClient = { connection }
+    const sql = `DROP ${typeOfElement} ${wrapIdentifier(elementName)}`
```

Comment:
`typeOfElement` needs wrapping as a literal here, and in all the other dialects, `${MysqlData.wrapLiteral(typeOfElement}`

Reply:
the `typeOfElement` was getting wrapped when passed into the function, but I get what you're saying. Handle it in each function so it does what it has to do closer to where it's being executed.